tests/JustSaying.Extensions.DependencyInjection.AwsCore.Tests/ClientFactoryTests.cs

```diff
@@ -31,76 +41,88 @@ public void ShouldReadConfigFromFile()
     [Fact]
     public void ShouldReadFromDefaultProfile()
     {
-        Environment.SetEnvironmentVariable("AWS_PROFILE", null);
-        var bin = AppContext.BaseDirectory;
-        Environment.SetEnvironmentVariable("AWS_CONFIG_FILE", $"{bin}/config");
-        Environment.SetEnvironmentVariable("AWS_SHARED_CREDENTIALS_FILE", $"{bin}/credentials");
+        //Arrange
         var config = new ConfigurationBuilder()
             .AddEnvironmentVariables()
             .Build();
         var serviceCollection = new ServiceCollection();
-        serviceCollection.AddJustSyaingWithAwsConfig(config, (_) => { });
+        serviceCollection.AddJustSayingWithAwsConfig(config, (_) => { });
+
+        //Act
         var provider = serviceCollection.BuildServiceProvider();
         var clientFactory = provider.GetRequiredService<IAwsClientFactory>();
-        var snsClient = clientFactory.GetSnsClient(RegionEndpoint.EUWest1);
+        var snsClient = clientFactory.GetSnsClient(RegionEndpoint.EUWest2);
+
+        //Assert
         Assert.True(string.IsNullOrEmpty(snsClient.Config.ServiceURL), snsClient.Config.ServiceURL);
-        Assert.True(snsClient.Config.RegionEndpoint == RegionEndpoint.EUWest1);
+        Assert.True(snsClient.Config.RegionEndpoint == RegionEndpoint.EUWest2);
         Assert.False(snsClient.Config.UseHttp);
```

Comment:
Use Shouldy for better assert messages in all these tests (that's what the other tests use).

Also, always use _"Equals(a, b)"_ type constructs not _"True(a == b)"_ ones as you get better error messages if the test fails (_"Expected a but got b"_ instead of _"Expected true but got false"_).

So, for example:

```csharp
snsClient.Config.ServiceURL.ShouldBeNull(); // or empty, whichever is the correct expectation
snsClient.Config.RegionEndpoint.ShouldBe(RegionEndpoint.EUWest2);
```